microprofile/config/config/src/main/java/io/helidon/microprofile/config/MpConfigBuilder.java

```diff
@@ -376,8 +391,9 @@ public String description() {
             return configSource.changes();
         }
 
-        int getOrdinal() {
-            return ordinal;
+        @Override
+        public int priority() {
+            return 100 - ordinal;
```

Comment:
I am kind of confused why is it 100 - ordinal?

Reply:
This is an aribtrary constant used to get from ordinals to priority.
Sources with ordinals will in most cases (unless negative priority is used) win over sources with Priority.
This is undefined in any specification, as MP uses ordinals, and Helidon uses Priority.
Mixing Helidon config sources with MP config sources is not recommended and not documented as an option.